tracker/view/blueprint.py

```diff
@@ -67,3 +68,9 @@ def urlize(ctx, text, trim_url_limit=None, rel=None, target=None):
 
     text = do_urlize(ctx, text, trim_url_limit=trim_url_limit, target=target, rel=rel)
     return text
+
+
+@blueprint.app_template_filter()
+def diff(previous, current):
+    return ndiff(str(previous).splitlines() if previous is not None else '',
+                 str(current).splitlines() if current is not None else '')
```

Comment:
```python
@blueprint.app_template_filter()
def diff(previous, current):
    return ndiff(str(previous).splitlines() if previous else '',
                 str(current).splitlines() if current else '')
```
Is there any reason why this wouldn't work here?

Reply:
yes, because we want to diff on `True` and `False` and just assume no value if we have a `None` type as `False` evaluates otherwise to empty string